tests/generation/FileGlobs.py

```diff
@@ -91,8 +89,8 @@ def test_get_project_files_gitignore(self):
         with suppress_stdout():
             self.assertEqual(
                 sorted(get_project_files(
-                    self.log_printer, self.printer, os.getcwd())[0]),
-                sorted(files))
+                    self.log_printer, self.printer, os.getcwd(),
+                self.noninteractive)[0]), sorted(files))
```

Comment:
@adtac You mean don't touch at all this test? But when I do a pytest I get " missing 1 required positional argument", because my function has 4 arguments.